include/text.php

```diff
@@ -933,12 +933,12 @@ function contact_block() {
 	$a = get_app();
 
 	$shown = get_pconfig($a->profile['uid'],'system','display_friend_count');
-	if($shown === false)
+	if ($shown === false)
```

Comment:
I stopped noticing all the remaining violations because @Quix0r's process isn't exhaustive. I offered him to improve it, but he never accepted, so right now I'm just making sure there aren't any Parse Error, and someone™️ will do a second pass. Either @Quix0r or me, it doesn't really matter.